packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -231,6 +233,8 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
 ///
 /// See also:
 ///
+///  * [RenderSliverPrototypeExtentList], which uses a prototype list item intead
+///    a pixel value, to define the extent of each item.
```

Comment:
there's many more examples of "intead a" below, all of which should be fixed too